README.md

```diff
@@ -8,7 +8,7 @@ Code samples and other handouts for our course. See the [**practice exercises**]
 
 ## Course Summary
 
-Learning to program can be overwhelming. Concepts and facts come at you fast and most courses don't cover many of them at all or at a beginner's pace. **This is not most courses**. Python for absolute beginners is our premier course for beginning developers. We start at the very beginning, teaching you the big ideas and concepts covered in a CS 101 course. Then we move on to writing increasingly complex code and applications in Python.
+Learning to program can be overwhelming. Concepts and facts come at you fast and most courses don't cover many of them at all or at a beginner's pace. **This is not like most courses**. Python for absolute beginners is our premier course for beginning developers. We start at the very beginning, teaching you the big ideas and concepts covered in a CS 101 course. Then we move on to writing increasingly complex code and applications in Python.
```

Comment:
Hi, I know it's not the most common way of speaking, but I did want it to say: **This is not most courses**, to emphasize what was said in the prior sentence.

Reply:
Alright,it's up to you.Thanks for review.